docs/installation.md

```diff
@@ -29,10 +29,11 @@ require_once '/path/to/qingstor-sdk.phar';
 
 ```bash
 git clone git@github.com:yunify/qingstor-sdk-php.git
+cd qingstor-sdk-php && composer install
 ```
 
 2. Require `autoload.php` in your php code
```

Comment:
phar-composer will use index.php for start, and it works well with changed to index.php